meta/.meta

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Why do we want to duplicate our component data? 
I see we already have https://github.com/opensearch-project/project-meta which takes care of this. 
Why not use it from opensearch-build for releases ?

Reply:
In meta we have all our repositories, not all components that are included in an OpenSearch release. I considered creating another folder in project-meta for just those, but effectively this list is the sum of OpenSearch + OpenSearch Dashboards manifests and is about what we release, which all lives in this repo.

Reply:
Ok I would rather have one single source of truth and may be add parameters. 

Anyway, how do we plan on to keep this up to date?
I assume its manual and for a component owner this should be part of onboarding(ONBOARDING.md) to release workflow.

Reply:
I agree one source of truth is what we want, and the truth is here ;)

This is manual for now indeed. I'll think of some automation that could ensure that what's in manifests matches this list.